src/modules/homepage/SpendFrequency.tsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Stack } from '@mui/material';
+import { ReactComponent as Graphic } from '../../assets/illustrations/graphic.svg';
+import { theme } from 'src/styles/theme';
+import { colors } from 'src/styles/colors';
+import { Title3 } from '../../shared-components/Typography/Title3';
+
+export const SpendFrequency = () => {
+  return (
+    <Stack>
+      <Box
+        padding={theme.spacing(2)}
+        display='flex'
```

Comment:
You don't need to make this element a flex container, just add `text-align: start`

Reply:
fixed